Codeinwp/neve
# fix: multiple embeds acting weird [closes #951]

### Summary
Fixes embeds display.


### Will affect visual aspect of the product

NO

### Test instructions


paste the list below into a gutenberg post - it should embed everything
```

https://www.amazon.com/gp/product/B07Y8HLS8T?storeType=ebooks&pf_rd_t=40901&pd_rd_i=B07Y8HLS8T&pf_rd_m=A2R2RITDJNW1Q6&pageType=STOREFRONT&pf_rd_p=ae642e4b-ad2c-4298-8dd0-2f96f1bbdbdd&pf_rd_r=D2JARDZ4GPC0JTC9VRH8&pd_rd_wg=IO0iB&pf_rd_s=merchandised-search-3&ref_=dbs_f_ebk_rwt_wigo_ts_ms3_kmw_ae642e4b-ad2c-4298-8dd0-2f96f1bbdb&pd_rd_w=GhbBM&pf_rd_i=154606011&pd_rd_r=acf4bc8f-fcf5-49b1-874a-0c13d2c92d79 
https://abaicus.tumblr.com/post/119261724008 
https://speakerdeck.com/morganepeng/design-and-strategy-how-to-deal-with-people-who-dont-get-design 
https://cloudup.com/iDYua017tIR 
https://www.reddit.com/r/Romania/comments/dxqnfc/violenta_domestica_din_bucuresti_anul_1888/ 
https://soundcloud.com/bosquito-official/bine 
https://vimeo.com/276012526 
https://www.flickr.com/photos/themeisle/48964084666/ 
https://democodecyclopedia.survey.fm/demo 
https://wordpress.tv/2019/08/09/catalin-matei-why-and-how-to-use-social-media-to-create-a-brand-and-revenue/ 
https://www.ted.com/talks/gaby_barrios_why_gender_based_marketing_is_bad_for_business 
https://www.slideshare.net/NVIDIA/top-5-deep-learning-and-ai-stories-october-6-2017-80543540 
https://www.scribd.com/document/1/Improved-Statistical-Test#page=4 
https://www.meetup.com/Bucharest-Java-User-Group/ 
https://www.mixcloud.com/spacemusicpodcast/spacemusic-1114-lifedeath-nonstop-edition/ 
https://kck.st/32TV1Gz 
https://www.dailymotion.com/video/x7nygfx?playlist=x5v2j4 
http://www.collegehumor.com/video/60123714/four-ways-to-actually-declutter-your-home 
https://animoto.com/play/n2sd7PFxfhFDlvd1tfK2Ig 
https://www.reverbnation.com/maxkhameleon5 
https://www.instagram.com/p/B1o0v5iAVyH/?utm_source=ig_web_copy_link 
https://open.spotify.com/playlist/5dwlVoN1GAFX3nYi0lTwn2 
https://www.youtube.com/watch?v=1aheRpmurAo 
https://wordpress.org/support/article/embeds/ 
```


Closes #951.
